package.json

```diff
@@ -123,6 +122,7 @@
   },
   "license": "GPL-3.0-or-later",
   "dependencies": {
-    "@gnosis.pm/gp-v2-contracts": "^0.0.1-alpha.10"
+    "@gnosis.pm/gp-v2-contracts": "^0.0.1-alpha.11",
+    "@uniswap/sdk": "npm:uniswap-xdai-sdk"
```

Comment:
Where can I see where this package comes from. The NPM package points to the original uniswap

is this something meant to be a permanent solution or was a one time for the DEMO thing?